### ISSUE TITLE
getting rejections on requests that are not restricted

### ISSUE THREAD

**Issue by abrin** [sdonk/django-admin-ip-restrictor#23]
I believe we're seeing a bug related to the current open pull-request whereby requests to non-restricted IPs are throwing exceptions because they're private IPs.  In the example below, our config is setup to just restrict access to ["admin"] yet, we're seeing exceptions related to the middleware

```
"GET /api/config HTTP/1.0" 500 27<socket.socket fd=6, family=AddressFamily.AF_INET, 
type=SocketKind.SOCK_STREAM, proto=0, laddr=('172.17.0.2', 8000), raddr=('172.17.0.3', 46444)>Traceback (most recent call last):
  File "/root/.local/share/virtualenvs/portal-lhpYkIEg/lib/python3.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.local/share/virtualenvs/portal-lhpYkIEg/lib/python3.7/site-packages/django/core/handlers/base.py", line 178, in _get_response
    response = middleware_method(request, callback, callback_args, callback_kwargs)
  File "/root/.local/share/virtualenvs/portal-lhpYkIEg/lib/python3.7/site-packages/admin_ip_restrictor/middleware.py", line 81, in process_view
    ip = self.get_ip(request)
  File "/root/.local/share/virtualenvs/portal-lhpYkIEg/lib/python3.7/site-packages/admin_ip_restrictor/middleware.py", line 76, in get_ip
    assert is_routable, 'IP is private'
AssertionError: IP is private 
```

**Comment by sdonk** [comment]
@abrin PR #24 is now merged and released to PyPi as version number 2.1.0. 
Please be aware that this version drops support for some versions of Python and Django

**Comment by abrin** [comment]
thanks for such a quick response, I'll give it a try
